pkg/cmd/pr/create/create.go

```diff
@@ -105,6 +105,14 @@ func NewCmdCreate(f *cmdutil.Factory, runF func(*CreateOptions) error) *cobra.Co
 			to push the branch and offer an option to fork the base repository. Use %[1]s--head%[1]s to
 			explicitly skip any forking or pushing behavior.
 
+      A prompt will also ask for the title and the body of the pull request.
+      Use %[1]s--title%[1]s and %[1]s--body%[1]s to skip this, or use
+      %[1]s--fill%[1]s to autofill these values from git commits. It's
+      important to notice that if the %[1]s--title%[1]s and/or %[1]s--body%[1]s
+      are also provided alongside %[1]s--fill%[1]s, the values specified by
+      %[1]s--title%[1]s and/or %[1]s--body%[1]s will take precedence and
+      overwrite any autofilled content.
```

Comment:
Small nit but can we get this text aligned with the other text in `Long`, looks like the spacing is off a bit.

Reply:
Yea, I saw this, do you know the width? I use 80.

Reply:
@peterramaldes : that is a really good question.  not completely sure myself, I thought I might help this by adding a small contribution to the PR myself 🙇